src/platform/user/profile/actions/mhv.js

```diff
@@ -22,7 +22,7 @@ export function fetchMHVAccount() {
 
     apiRequest(baseUrl)
       .then(({ data }) => dispatch({ type: FETCH_MHV_ACCOUNT_SUCCESS, data }))
-      .catch(({ errors }) =>
+      .catch(({ errors = {} }) =>
```

Comment:
This fixes an edge case I ran into when testing Profile 2.0. If the call to get MHV status fails but for some reason there is no `errors` from the failed response, then the value of `errors` in the Redux state would be `null` when it should really be a non-null value to indicate that there was an error loading the MHV status.